sdk/java-sdk/src/main/scala/kalix/javasdk/impl/AnySupport.scala

```diff
@@ -397,16 +381,8 @@ class AnySupport(
         ScalaPbAny(BytesPrimitive.fullName, primitiveToBytes(BytesPrimitive, byteString))
 
       case other =>
-        // last chance: try to find a suitable JacksonProto serializer
-        additionalSerializers
```

Comment:
this was added to support serialization using protobuf generated from Jackson lib. We don't need it anymore.